01_P/P_2_3_2_01/sketch.js

```diff
@@ -38,13 +39,11 @@ var x = 0, y = 0;
 var stepSize = 5.0;
 var lineLength = 25;
 
-
 function setup() {
   col = color(random(255),random(255),random(255),random(100));
-  // use full screen size 
+  // use full screen size
   createCanvas(displayWidth, displayHeight);
   background(255);
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).